packages/snaps-controllers/src/interface/SnapInterfaceController.ts

```diff
@@ -393,4 +438,17 @@ export class SnapInterfaceController extends BaseController<
       (id: string) => this.messagingSystem.call('SnapController:get', id),
     );
   }
+
+  /**
+   * Utility function to validate the type of interface content.
+   * Must be a value of the enum ContentType.
+   * Throws if the passed string is invalid.
+   *
+   * @param contentType - The content type.
+   */
+  #validateContentType(contentType: string) {
+    if (!(contentType in ContentType)) {
+      throw new Error('Invalid content type.');
```

Comment:
This should be tested

Reply:
Function is no longer needed after removing update content type function.